features/order_confirmation.feature

```diff
@@ -0,0 +1,12 @@
+Feature: As a Visitor.
+  In order to know when my order is ready for pick-up.
+  After clicking buy now, I would like to see an order confirmation with estimated pick up time.
```

Comment:
Fair enough, but I would still say "After finalizing my order" or something similar. Or at least clicking 'Buy Now' (btw I think that is bad UX) Not a blocker though. Let's move on...